models/location_helpers_test.go

```diff
@@ -80,10 +80,11 @@ func TestBackupLocations(t *testing.T) {
 			Description: "some desc",
 			BackupLocationConfig: models.BackupLocationConfig{
 				S3Config: &models.S3LocationConfig{
-					Endpoint:   "https://example.com/",
-					AccessKey:  "access_key",
-					SecretKey:  "secret_key",
-					BucketName: "example_bucket",
+					Endpoint:       "https://example.com/",
+					AccessKey:      "access_key",
+					SecretKey:      "secret_key",
+					BucketName:     "example_bucket",
+					BucketLocation: "us-east-2",
```

Comment:
it increases complexity of reading the code, maybe it's better to rename it to `region`. `BackupLocation`, `BucketLocation` looks very similar.